assets/blocks/course-outline/lesson-block/settings.js

```diff
@@ -27,45 +33,60 @@ export const LessonBlockSettings = ( {
 	);
 
 	return (
-		<InspectorControls>
-			{ id && (
-				<PanelBody title={ __( 'Lesson', 'sensei-lms' ) }>
-					<h2>
+		<>
+			<InspectorControls>
+				{ id && (
+					<PanelBody title={ __( 'Lesson', 'sensei-lms' ) }>
+						<h2>
+							<ExternalLink
+								href={ `post.php?post=${ id }&action=edit` }
+								target="lesson"
+								className="wp-block-sensei-lms-course-outline-lesson__edit"
+							>
+								{ __( 'Edit lesson', 'sensei-lms' ) }
+							</ExternalLink>
```

Comment:
Can we move this to an `editPostLink` variable and share with the settings panel to avoid code duplication?

Reply:
Good catch! https://github.com/Automattic/sensei/pull/3708/commits/c4db8a6d252e8589cc23814540cd211d89c7313f